devnet-azero/devnet-azero-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.9'
+
+services:
+  # key derived from "//0"
+  AlephBridgenode:
+    image: public.ecr.aws/p6e8q1z1/aleph-node:r-12.1
+    container_name: aleph_bridgenode
```

Comment:
you might want to rename it (to exclude 'bridge' part); here and in the 26th line